debian/control

```diff
@@ -16,8 +16,8 @@ Build-Depends: debhelper (>= 11~),
                protobuf-compiler
 Standards-Version: 4.1.4
 Homepage: https://github.com/opencontainers/runc
-Vcs-Git: https://salsa.debian.org/go-team/packages/runc.git
-Vcs-Browser: https://salsa.debian.org/go-team/packages/runc
+Vcs-Git: https://github.com/tianon/debian-runc.git
+Vcs-Browser: https://github.com/tianon/debian-runc
```

Comment:
This should probably include the branch, right?

```suggestion
Vcs-Git: https://github.com/tianon/debian-runc.git -b ubuntu
Vcs-Browser: https://github.com/tianon/debian-runc/tree/ubuntu
```

Reply:
Thanks for catching this up. Fixed.